csunplugged/topics/content/en/binary-numbers/unit-plan/unit-plan.yaml

```diff
@@ -26,8 +26,8 @@ how-binary-digits-work:
     - binary-correct-representation
     - no-physical-zeros-ones
   generated-resources:
-    sorting-network:
-      description: One per student.
+    binary-cards:
+      description: One set.
```

Comment:
does it make sense to move the description into the `resources.yaml` Or will there be a chance that one resource could have different descriptions?